src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://www.google.com/" 
+      method="post" 
+      data-qa="big"
+    >
+      <input 
```

Comment:
Every input must have own label. Pay attention, please, on this point in future

Reply:
+